Essentials/src/main/java/com/earth2me/essentials/MobCompat.java

```diff
@@ -175,6 +177,24 @@ public static void setFrogVariant(final Entity entity, final String variant) {
         }
     }
 
+    public static void setBoatVariant(final Entity entity, final BoatVariant variant) {
+        if (VersionUtil.getServerBukkitVersion().isLowerThan(VersionUtil.v1_9_R01)) {
+            return;
+        }
+        final Boat boat;
+        if (entity instanceof Boat) {
+            boat = (Boat) entity;
+        } else {
+            return;
+        }
+        if (VersionUtil.getServerBukkitVersion().isLowerThan(VersionUtil.v1_19_R01)) {
```

Comment:
This is always false because of the check at the head of the method. Is the check at the head a mistake?

Reply:
The version can be lower than 1.19 without being lower than 1.9

Reply:
i'm dumb